src/client/registry.ts

```diff
@@ -85,6 +88,26 @@ const bindResponseValidator = zod.object({
   map_ast: zod.string(),
 });
 
+export function loadSdkAuthToken(): string | undefined {
+  const tokenEnvName = 'SUPERFACE_SDK_TOKEN';
+  //Load superface token
+  const loadedToken = process.env[tokenEnvName];
+  if (!loadedToken) {
+    registryDebug(`Environment variable ${tokenEnvName} not found`)
+
+    return;
+  }
+  const token = loadedToken.trim();
+  const tokenRegexp = /^(sfs)_(.+)_([0-9A-F]{8}$)/i;
```

Comment:
Greedy regex parser goes brrr. Pls replace `.` with `[^_]` at least.

Reply:
Also the `$` doesn't need to be in the last group

Reply:
I just reused regex from BE 🤨🤷